src/main/pg/sbus_output.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file is part of Rotorflight.
+ *
+ * Rotorflight is free software. You can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Rotorflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "common/utils.h"
+#include "pg/pg.h"
+
+#define SBUS_OUT_CHANNELS 18
+
+typedef enum {
+    SBUS_OUT_SOURCE_RX = 0,
```

Comment:
Please add SBUS_OUT_SOURCE_NONE = 0, for nothing on the channel

Reply:
Added